qa/L0_config_search/test.sh

```diff
@@ -122,6 +142,18 @@ for launch_mode in $TRITON_LAUNCH_MODES; do
             RET=1
         fi
 
+        # Check that GPUs don't match and we return an error
+        if [ $launch_mode != 'remote' ]; then
```

Comment:
why not do this for remote?

Reply:
Would have been more code complexity to get this to work with keeping the remote server up and then shutting it down correctly.

And, I only need to cover this case for one (non-c_api) mode to exercise this code.